packages/dropdowns/src/types/index.ts

```diff
@@ -286,10 +286,14 @@ export interface IItemProps extends Omit<LiHTMLAttributes<HTMLLIElement>, 'value
   icon?: ReactElement;
   /** Indicates that the item is not interactive */
   isDisabled?: boolean;
+  /** If the item is an anchor, opens the link externally */
```

Comment:
```suggestion
  /** Opens the `href` externally */
```

Follow [documentation grammar guidelines](https://github.com/zendeskgarden/react-components/blob/main/docs/documentation.md).